go.mod

```diff
@@ -32,6 +32,7 @@ require (
 	google.golang.org/protobuf v1.27.1
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
+	internal/idocs v1.0.0
```

Comment:
Any reason for creating a separate module for docs?

Reply:
We should only export APIs that our users need. `internals` makes the code an internal package, so it's not accessible to anyone else and is no longer part of the public API. I'm writing a doc and will share with you. It's also useful for versioning.

Reply:
Yes. I agree.

Reply:
Isn't naming the dir as `internal` enough to avoid exposing this as public API? Does it enforce that this be a new go module?